app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)

```

Comment:
Code Style issue: You need to use one style of quotes in your code. Double quotes are preferable. Here you are using both single and double quotes.